pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: flutter_home_connect_sdk
+description: A starting point for Dart libraries or applications.
+version: 1.0.0
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.18.2 <3.0.0'
+
+# dependencies:
+#   path: ^1.8.0
+
```

Comment:
Should we add `build_runner`, `json_serializable`, and `json_annotation` ?

Reply:
we can add those when we start using them